packages/unmock-core/src/backend/index.ts

```diff
@@ -124,17 +101,12 @@ export default class NodeBackend {
     }
 
     const createResponse = responseCreatorFactory({
-      listeners: [
-        new FSLogger({
-          directory: this.config.servicesDirectory,
-        }),
-        FSSnapshotter.getOrUpdateSnapshotter({}),
-      ],
+      listeners: this.listeners,
       options,
       store: this.serviceStore,
     });
 
-    this.interceptor = createInterceptor({
+    this.interceptor = new this.InterceptorCls({
```

Comment:
Not very critical, but it feels weird to call `createResponse` a listener. Is it the same kind of listener as `this.listeners`?

Reply:
`listeners` was renamed to "onSerializedRequest" in #307 so I'll have to merge that here.